dev/com.ibm.websphere.appserver.features/visibility/private/io.openliberty.netty.internal.feature

```diff
@@ -0,0 +1,8 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=io.openliberty.netty.internal
+WLP-DisableAllFeatures-OnConflict: false
+singleton=true
+-bundles=io.openliberty.netty.internal; location:="lib/";
+kind=noship
+edition=core
```

Comment:
edition=full until the feature is moved to beta

Reply:
I would prefer to leave it at the correct value. Why do you care?

Reply:
As discussed, I believe that is just a example, not guidance,  in the doc. Without some reason, I am happy to test leaving it correct from the start.